graphlearn/examples/tf/trainer.py

```diff
@@ -38,141 +39,180 @@
 run_options = tf.RunOptions(trace_level=tf.RunOptions.FULL_TRACE)
 run_metadata = tf.RunMetadata()
 
-class DistTrainer(object):
-  """Class for distributed training and evaluation
+class TFTrainer(object):
+  """Class for local or distributed training and evaluation.
 
   Args:
-    cluster_spec: TensorFlow ClusterSpec.
-    job_name: name of this worker.
     task_index: index of this worker.
```

Comment:
rm this unused arg.

Reply:
Docstring has been revised in trainer.py.